repositories/geometry2.BUILD.bazel

```diff
@@ -158,6 +158,19 @@ ros2_cpp_binary(
     ],
 )
 
+ros2_cpp_library(
+    name = "cpp_tf2_geometry_msgs",
+    hdrs = ["tf2_geometry_msgs/include/tf2_geometry_msgs/tf2_geometry_msgs.hpp"],
```

Comment:
Should we also include the corresponding .h file, just in case someone would still use it?

Reply:
I'll leave that up to you. I left it out because it's long deprecated and other similar libraries in rules_ros2 didn't include the .h file.

Reply:
OK, let's leave it as-is and add if necessary.